corehq/apps/app_manager/views/formdesigner.py

```diff
@@ -53,216 +52,77 @@
 
 @require_can_edit_apps
 def form_designer(request, domain, app_id, module_id=None, form_id=None):
-
-    def _form_uses_case(module, form):
-        return (
-            (module and module.case_type and form.requires_case()) or
-            is_usercase_in_use(domain)
-        )
-
-    def _form_is_basic(form):
-        return form.doc_type == 'Form'
-
-    def _form_too_large(app, form):
-        # form less than 0.1MB, anything larger starts to have
-        # performance issues with fullstory
-        return app.blobs['{}.xml'.format(form.unique_id)]['content_length'] > 102400
-
-    meta = get_meta(request)
-    track_entered_form_builder_on_hubspot.delay(request.couch_user, request.COOKIES, meta)
-
     app = get_app(domain, app_id)
-    module = None
 
     try:
         module = app.get_module(module_id)
     except ModuleNotFoundException:
         return bail(request, domain, app_id, not_found="module")
+
     try:
         form = module.get_form(form_id)
-    except IndexError:
+    except AttributeError:
         return bail(request, domain, app_id, not_found="form")
 
+    return _get_form_designer_view(request, domain, app, module, form)
+
+
+def _get_form_designer_view(request, domain, app, module, form):
     if form.no_vellum:
         messages.warning(request, _(
             "You tried to edit this form in the Form Builder. "
             "However, your administrator has locked this form against editing "
             "in the form builder, so we have redirected you to "
             "the form's front page instead."
         ))
-        return back_to_main(request, domain, app_id=app_id,
+        return back_to_main(request, domain, app_id=app.id,
                             form_unique_id=form.unique_id)
 
-    include_fullstory = False
-    vellum_plugins = ["modeliteration", "itemset", "atwho"]
-    if (toggles.COMMTRACK.enabled(domain)):
-        vellum_plugins.append("commtrack")
-    if toggles.VELLUM_SAVE_TO_CASE.enabled(domain):
-        vellum_plugins.append("saveToCase")
-    if (_form_uses_case(module, form) and _form_is_basic(form)):
-        vellum_plugins.append("databrowser")
-
-    vellum_features = toggles.toggles_dict(username=request.user.username, domain=domain)
-    vellum_features.update({'advanced_itemsets': add_ons.show("advanced_itemsets", request, app)})
```

Comment:
I think this got lost in the refactor, it doesn't appear in `_get_vellum_features`